events/events.go

```diff
@@ -0,0 +1,308 @@
+package events
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/service-fabrik-cli-plugin/errors"
+	"github.com/SAP/service-fabrik-cli-plugin/helper"
+	"github.com/cloudfoundry/cli/plugin"
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
```

Comment:
Are there any extra dependencies added here which were not already present?

Reply:
I think no.